R/data_type.R

```diff
@@ -130,25 +132,25 @@ identify_data_type <- function(df,
 
 #' Split raw data into list based on the data types
 #'
-#' @param df data.frame of raw drug response data containing both treated and 
+#' @param df data.table of raw drug response data containing both treated and 
 #' untreated values with column specified in `type_col` argument.
 #' @param type_col string with column names in `df` with info about data type.
 #' Defaults to \code{"type"}.
 #' 
 #' @examples 
 #' cell_lines <- gDRtestData::create_synthetic_cell_lines()
 #' drugs <- gDRtestData::create_synthetic_drugs()
-#' df_layout <- merge(cell_lines[7:8, ], drugs[c(4:6), ], by = NULL)
+#' df_layout <- data.table::setDT(merge.data.frame(cell_lines[7:8, ], drugs[c(4:6), ], by = NULL))
```

Comment:
Please do not use `data.table::setDT` in assignment.
You can simplify this line by: `df_layout <-  drugs[7:8, as.list(cell_lines[4:6, ]), names(drugs)]`